.circleci/config.yml

```diff
@@ -99,6 +99,10 @@ commands:
               echo "There are git differences after running yarn install"
               exit 1
             fi
+      - run:
+          name: Fail if there are vulnerabilities in packages
+          command: |
+            /bin/bash -c 'yarn audit --groups dependencies --level high; [[ $? -ge 8 ]] && exit 1 || exit 0'
```

Comment:
Does `yarn audit` work for you? It always fails for me cause our monorepo is too big. Maybe that's been fixed